.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/qlik-date-picker
   docker:
-    - image: circleci/node:stretch
+    - image: circleci/node:11.15.0
```

Comment:
You need to rebase against the qlik-date-picker branch. I've already fixed this problem by making the extension build with gulp 4.

Reply:
Shall do that !